diff --git a/lib/FuzzyJava9Lexer.js b/lib/FuzzyJava9Lexer.js
index ec6ad21..ee82957 100644
--- a/lib/FuzzyJava9Lexer.js
+++ b/lib/FuzzyJava9Lexer.js
@@ -1039,7 +1039,7 @@ FuzzyJava9Lexer.prototype.JavaLetter_sempred = function(localctx, predIndex) {
 		case 0:
 			return Character.isJavaIdentifierStart(_input.LA(-1));
 		case 1:
-			return Character.isJavaIdentifierStart(Character.toCodePoint((char)_input.LA(-2), (char)_input.LA(-1)));
+			return Character.isJavaIdentifierStart(Character.toCodePoint(_input.LA(-2), _input.LA(-1)));
 		default:
 			throw "No predicate with index:" + predIndex;
 	}
@@ -1050,7 +1050,7 @@ FuzzyJava9Lexer.prototype.JavaLetterOrDigit_sempred = function(localctx, predInd
 		case 2:
 			return Character.isJavaIdentifierPart(_input.LA(-1));
 		case 3:
-			return Character.isJavaIdentifierPart(Character.toCodePoint((char)_input.LA(-2), (char)_input.LA(-1)));
+			return Character.isJavaIdentifierPart(Character.toCodePoint(_input.LA(-2), _input.LA(-1)));
 		default:
 			throw "No predicate with index:" + predIndex;
 	}
